docs/install.rst

```diff
@@ -56,17 +56,32 @@ OS X + Docker
 Ubuntu 14.04 (manual way)
 -------------------------
 
-1. Install system dependencies (check
-`provision.sh <https://github.com/scrapinghub/splash/blob/master/dockerfiles/splash/provision.sh>`)
-
-2. Clone the repo from GitHub::
+1. Clone the repo from GitHub::
 
         $ git clone https://github.com/scrapinghub/splash/
 
-3. Install dependencies with pip::
+2. Install dependencies::
+
+        $ cd splash/dockerfiles/splash   
+        
+        $ sudo ./provision.sh prepare_install 
+        
+        $ sudo ./provision.sh install_msfonts 
+        
+        $ sudo ./provision.sh install_builddeps
+        
+        $ sudo ./provision.sh install_deps
+        
+        $ sudo ./provision.sh install_extra_fonts
+       
+        $ sudo ./provision.sh install_pyqt5
+       
+        $ sudo ./provision.sh install_python_deps
```

Comment:
Oops, `install_flash` was omitted by mistake. 

I haven't tested it, but from looking at the script it looks like it will properly operate via:
`sudo ./provision.sh prepare_install install_msfonts install_builddeps install_deps etc..`

That is certainly more elegant, is that what you mean?